components/applications-service/pkg/wrap/wrap_test.go

```diff
@@ -0,0 +1,50 @@
+package wrap
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"github.com/chef/automate/api/external/applications"
+)
+
+func TestWrap(t *testing.T) {
+	ex1 := applications.ExampleOne{
+		IntOne: 23,
+		IntTwo: 42,
+	}
+
+	packedEx1, err := proto.Marshal(&ex1)
```

Comment:
Even though we have to "manually" marshal and unmarshal the nested protobufs, it's a similar amount of code because when we used `protobuf.Any` we needed to call `ptypes.MarshalAny()` and `ptypes.UnmarshalAny()`